q2_feature_table/_summarize/_visualizer.py

```diff
@@ -47,42 +46,40 @@ def view_taxa_data(output_dir: str, data: pd.Series) -> None:
 
 
 def summarize(output_dir: str, table: biom.Table) -> None:
+    number_of_samples = len(table.ids(axis='sample'))
+    number_of_features = len(table.ids(axis='observation'))
+
     sample_summary, sample_counts = _count_summary(table, axis='sample')
+    if number_of_samples > 1:
+        sample_counts_ax = sns.distplot(sample_counts, kde=False, rug=True)
+        sample_counts_ax.set_xlabel('Frequency of sample')
```

Comment:
"Frequency of sample" is a little confusing, it seems to imply the number of times a sample was observed. Do you think "Frequency per sample" makes sense? That title would also match the table titles you're using.

Same question about "Frequency of feature" below.